test/adapter/json_api/relationship_test.rb

```diff
@@ -27,7 +27,6 @@ class RelationshipAuthorSerializer < ActiveModel::Serializer
             has_many :posts do
               link :related do
                 ids = object.posts.map!(&:id).join(',')
-                href "//example.com/posts/#{ids}"
```

Comment:
In #1504, I originally wrote the test to test the `meta` call and there was no call to `href`. But then found out that we had this bug and added a call to `href` to prevent having a `nil` href. This change simply remove the `href` to test the behavior I wanted to test from the beginning. Thinking about it again,  I could also had another test instead.